---
title: "Comprehensive mapping of cystic fibrosis mutations to CFTR protein identifies mutation clusters and molecular docking predicts corrector binding site"
collection: publications
category: manuscripts
permalink: /publication/2018-04-10-cftr
excerpt: 'Cystic Fibrosis (CF) is caused by mutations in the CFTR gene, of which over 2000 have been reported to date. Mutations have yet to be analyzed in aggregate to assess their distribution across the tertiary structure of the CFTR protein, an approach that could provide valuable insights into the structure-function relationship of CFTR. In addition, the binding site of Class I correctors (VX-809, VX-661, and C18) is not well understood. In this study, exonic CFTR mutations and mutant allele frequencies described in 3 curated databases (ABCMdb, CFTR1, and CFTR2, comprising >130 000 data points) were mapped to 2 different structural models: a homology model of full-length CFTR protein in the open-channel state, and a cryo-electron microscopy core-structure of CFTR in the closed-channel state. Accordingly, residue positions of 6 high-frequency mutant CFTR alleles were found to spatially co-localize in CFTR protein, and a significant cluster was identified at the NBD1:ICL4 interdomain interface. In addition, immunoblotting confirmed the approximate binding site of Class I correctors, demonstrating that these small molecules act via a similar mechanism in vitro, and in silico molecular docking generated binding poses for their complex with the cryo-electron microscopy structure to suggest the putative corrector binding site is a multi-domain pocket near residues F374-L375. These results confirm the significance of interdomain interfaces as susceptible to disruptive mutation, and identify a putative corrector binding site. The structural pharmacogenomics approach of mapping mutation databases to protein models shows promise for facilitating drug discovery and personalized medicine for monogenetic diseases.'
date: 2018-04-10
venue: "Proteins"
paperurl: "https://doi.org/10.1002/prot.25496" 
citation: "Steven V. Molinski, Vijay M. Shahani, Adithya S. Subramanian, Stephen S. MacKinnon, Geoffrey Woollard, Marcon Laforet, Onofrio Laselva, Leonard D. Morayniss, Christine E. Bear, Andreas Windemuth. (2018). &quot;Comprehensive mapping of cystic fibrosis mutations to CFTR protein identifies mutation clusters and molecular docking predicts corrector binding site.&quot; <i>Proteins</i>. 86(8):833-843."
---

Cystic Fibrosis (CF) is caused by mutations in the CFTR gene, of which over 2000 have been reported to date. Mutations have yet to be analyzed in aggregate to assess their distribution across the tertiary structure of the CFTR protein, an approach that could provide valuable insights into the structure-function relationship of CFTR. In addition, the binding site of Class I correctors (VX-809, VX-661, and C18) is not well understood. In this study, exonic CFTR mutations and mutant allele frequencies described in 3 curated databases (ABCMdb, CFTR1, and CFTR2, comprising >130 000 data points) were mapped to 2 different structural models: a homology model of full-length CFTR protein in the open-channel state, and a cryo-electron microscopy core-structure of CFTR in the closed-channel state. Accordingly, residue positions of 6 high-frequency mutant CFTR alleles were found to spatially co-localize in CFTR protein, and a significant cluster was identified at the NBD1:ICL4 interdomain interface. In addition, immunoblotting confirmed the approximate binding site of Class I correctors, demonstrating that these small molecules act via a similar mechanism in vitro, and in silico molecular docking generated binding poses for their complex with the cryo-electron microscopy structure to suggest the putative corrector binding site is a multi-domain pocket near residues F374-L375. These results confirm the significance of interdomain interfaces as susceptible to disruptive mutation, and identify a putative corrector binding site. The structural pharmacogenomics approach of mapping mutation databases to protein models shows promise for facilitating drug discovery and personalized medicine for monogenetic diseases.